src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Delete.cs

```diff
@@ -78,6 +128,16 @@ public static partial class HttpClientJsonExtensions
             return DeleteFromJsonAsyncCore(taskResponse, type, context, cancellationToken);
         }
 
+        /// <summary>
+        /// Sends a DELETE request to the specified Uri and returns the value that results from deserializing the response body as JSON in an asynchronous operation.
+        /// </summary>
+        /// <param name="client">The client used to send the request.</param>
+        /// <param name="requestUri">The Uri the request is sent to.</param>
+        /// <param name="type">The type of the object to deserialize to and return.</param>
+        /// <param name="context">Source generated JsonSerializerContext used to control the deserialization behavior.</param>
```

Comment:
```suggestion
        /// <param name="context">Source generated JSON serializer context used to control the deserialization behavior.</param>
```

Reply:
Same applies to this snippet: https://github.com/dotnet/runtime/pull/74770#discussion_r957655108